R/url_path.R

```diff
@@ -0,0 +1,20 @@
+#' Construct URL Path
+#' 
+#' Construct a URL from path components like \code{file.path}
+#'
+#' @param x a URL, or vector of URLs
+#' @param ... character vectors
+#' 
+#' @examples
+#' 
+#' url_path("https://github.com", c('Ironholds', 'nfultz'), 'urltools')
+#' 
+#' @export
+#'
+url_path <- function(x, ...) {
+  p <- path(x)
+  p[is.na(p)] = '.'
```

Comment:
would this use `.` as a prefix for URLs? Does that actually work? Even `./` in the context of file systems is a unix-specific thing.